### ISSUE TITLE
performance

### ISSUE THREAD

**Issue by darkzbaron** [nicholasjconn/python-always-listening#1]
Hi there, did you get descent performance with this? I can't get your test to work!

**Comment by nicholasjconn** [comment]
I have had success with this script. Often, the quality of the microphone and recorded audio impacts the ability of the script to work. Have you recorded audio and played it back to see how it sounds?

**Comment by darkzbaron** [comment]
No, I think I will do some serious testing this week end. Good to hear it works! Thanks for putting that together.

Kind regards, Jonathan
